test/framework/provider/wait.go

```diff
@@ -84,12 +101,17 @@ func WaitForRevisionTransition(ctx context.Context, c client.Client, p2 string,
 }
 
 // Wait for Provider to be successfully deleted.
-func WaitForAllProvidersDeleted(ctx context.Context, c client.Client, interval time.Duration, timeout time.Duration) error {
+func WaitForAllProvidersDeleted(ctx context.Context, c client.Client, interval time.Duration, timeout time.Duration, t *testing.T) error {
 	return wait.PollImmediate(interval, timeout, func() (bool, error) {
 		l := &v1.ProviderList{}
 		if err := c.List(ctx, l); err != nil {
 			return false, err
 		}
+		for _, item := range l.Items {
+			t.Log("Undeleted providers :")
+			t.Logf("Name: %v \t Type: %v", item.Name, item.Kind)
```

Comment:
`item.Spec.Package` would be useful to log here also

Reply:
Do you mean logging `item.Spec.Package` in addition to the `item.Name` and `item.Kind`? @jbw976